internal/stack/parselogs.go

```diff
@@ -41,7 +42,8 @@ func ParseLogs(options ParseLogsOptions, process func(log LogLine) error) error
 		messageSlice := strings.SplitN(line, "|", 2)
 
 		if len(messageSlice) != 2 {
-			return fmt.Errorf("malformed docker-compose log line")
+			logger.Debugf("skipped malformed docker-compose log line: %s", line)
```

Comment:
Should we check if the malformed message is actually "Attaching to..." to avoid silently ignoring messages here?

Reply:
It could be checked that to narrow the skipped messages.
So for any other unexpected message, should we return error as it was before? @jsoriano

Reply:
What is the concern with missing unknown log format lines? ISTM that given that the vetting done later depends on knowing what we are looking for, unknown things will always be at risk of being missed.